src/react_native/core.cljs

```diff
@@ -134,26 +133,50 @@
 
 (def use-context react/useContext)
 
+(defn use-ref-atom
+  [value]
+  (let [ref (use-ref (atom value))]
+    (.-current ^js ref)))
+
+(defn get-js-deps
```

Comment:
`get-js-deps` is a misleading name, because in fact it's not really returning only JS deps, it's actually just wrapping Clojure data structures in a JS array. I don't fully understand this function tbh, a docstring might be a good idea too.

Reply:
if doest wrap, but it returns a kinda fake js-deps, it's just only one signal value, but it's true js, not a wrapper

Reply:
I meant wrapper in the sense that the return value is a JS array wrapping over CLJS data, but the elements of the array are not JS data structures. That's what's confusing to me on the name of the function.